lego/apps/events/templates/events/email/admin_registration.txt

```diff
@@ -8,7 +8,6 @@ En administrator har meldt deg på arrangementet {{ event }} på grunn av:
 
 {{ reason }}
 
-
-Les mer på {{ frontend_url }}/events/{{ id }}
+Ta kontakt med {{ creator }} dersom du skulle være uenig i dette.
```

Comment:
`Ta kontakt med {{ creator }} på {{ creator.email }} dersom du skulle være uenig i dette?` e.l.? Noe kontaktinfo kan være lurt. Evt. om vi klarer å finne komiteen til brukeren og legge ved den mailen 😛

Reply:
Nå så er vel ikke eventer bundet opp til gruppe, det må vel eventuelt fikses først... Er vel ikke så lurt å eventuelt prøve å finne ut av dette indirekte pga interessegrupper osv..

Reply:
Might as well just add the email. Users can be in multiple committees, so its trickier to get the committee.

Reply:
Also, it doesn't seem like the creator is sent in the context in `generate_mail()`?